2022/_data/jobs.yml

```diff
@@ -374,5 +374,109 @@
+        "link": "https://www.youtube.com/watch?v=-4digbYIFxs&list=PLKrSWEllR0Lg5c0eYT5AEi0vamlAZHilb"
+      },
+      {
+        "title": "Requirements Engineer",
+        "location": "Remote in Spain/ Onsite BCN/Vigo",
+        "description": "We are growing our Requirements Engineering / IT Business Analyst Team in Barcelona and Vigo and are looking for new colleagues who will bring their consultative and analytical experience spiced up by technical skills.<br/><br/><strong>Your Tasks</strong><br/>- You will be the link between our customer and our development teams and translate business requests to technical requirements and into technical concepts.<br/>- You will write business requirements, testing software, executing operational tasks and providing project management support while strengthening your technical acumen and business consulting skills. <br/>- You will take responsibility for the conception and design of web-based solutions in the digital marketing domain, designing, testing and launching the technology that meets fully the business need. You will truly be a game changer for our clients.<br/>- You will support the analysis and implementation of requirements along the entire software development cycle.<br/><br/><strong>What we require:</strong><br/>- Strong verbal and written English skills (C1 and higher)<br/>- First hands-on experience with web-based content management systems (CMS)<br/>- Solid technical understanding of web development and web application integrations<br/>- Basic knowledge of HTML / CSS / JavaScript and / or web-relevant programming languages<br/>- Experience in the creation of wireframes / mock-ups for IT applications<br/>",
```

Comment:
Spell checker reports:

* `meets fully the business need` should be  `fully meets the business needs`